gravitee-am-reporter/gravitee-am-reporter-kafka/src/main/java/io/gravitee/am/reporter/kafka/dto/AuditEntityDto.java

```diff
@@ -15,60 +15,30 @@
  */
 package io.gravitee.am.reporter.kafka.dto;
 
-public class AuditEntityDto {
-  private String id;
-  private String alternativeId;
-  private String type;
-  private String displayName;
-  private String referenceType;
-  private String referenceId;
-
-  public String getId() {
-    return id;
-  }
-
-  public void setId(String id) {
-    this.id = id;
-  }
-
-  public String getAlternativeId() {
-    return alternativeId;
-  }
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+import lombok.Setter;
 
-  public void setAlternativeId(String alternativeId) {
-    this.alternativeId = alternativeId;
-  }
-
-  public String getType() {
-    return type;
-  }
-
-  public void setType(String type) {
-    this.type = type;
-  }
+@Setter
+@Getter
+public class AuditEntityDto {
 
-  public String getDisplayName() {
-    return displayName;
-  }
+    @JsonProperty
```

Comment:
not sure it's needed (see [AuditAccessPointDto](https://github.com/gravitee-io/gravitee-access-management/pull/4268/files#diff-18abfb814783921c68a3df61da906d61e0ec78f51569d2eda90cd70b7b12d2dc))